maat-court-data-api/src/main/java/gov/uk/courtdata/entity/CorrespondenceEntity.java

```diff
@@ -0,0 +1,50 @@
+package gov.uk.courtdata.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.time.LocalDateTime;
+
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Data
```

Comment:
Please use @Getter and @Setter instead of @Data, as using the @DaTa annotation can cause performance issues when used on JPA entities.